index.html

```diff
@@ -10,229 +10,233 @@
+                </li>
+            </ul>
         </section>
 
-        <section class="card">
-            <section class="screenshot">
-                <img src="/Images/placeholder.png" alt="screenshot">
-            </section>
-            
-            <section class="info">
-                <h3 class="text">
-                    Multi-Post Stories Gain+Glory
-                </h3>
-                <ul class="languages">
-                    <li>HTML</li>
-                    <li>CSS</li>
-                    <li>Javascript</li>
-                </ul>
-                <a class="button" href="">See project</a>
-            </section>
-        </section>
 
-        <section class="card">
-            <section class="screenshot">
-                <img src="/Images/placeholder.png" alt="screenshot">
+        <section class="works">
+            <section id="works-title">
+                <h2>My recent works</h2>
+                <img src="/Images/works-horizontal-line.svg" alt="line">
             </section>
             
-            <section class="info">
-                <h3 class="text">
-                    Multi-Post Stories Gain+Glory
-                </h3>
-                <ul class="languages">
-                    <li>HTML</li>
-                    <li>CSS</li>
-                </ul>
-                <a class="button" href="">See project</a>
-            </section>
-        </section>
+            <div class="works">
+                <section class="card">
+                    <section class="screenshot">
+                        <img src="/Images/placeholder.png" alt="screenshot">
+                    </section>
+                    
+                    <section class="info">
+                        <h3 class="text">
+                            Multi-Post Stories Gain+Glory
+                        </h3>
+                        <ul class="languages">
+                            <li>HTML</li>
+                            <li>CSS</li>
+                        </ul>
+                        <a class="button" href="">See project</a>
+                    </section>
+                </section>
+    
+                <section class="card">
+                    <section class="screenshot">
+                        <img src="/Images/placeholder.png" alt="screenshot">
+                    </section>
+                    
+                    <section class="info">
+                        <h3 class="text">
+                            Multi-Post Stories Gain+Glory
+                        </h3>
+                        <ul class="languages">
+                            <li>HTML</li>
+                            <li>CSS</li>
+                            <li>Javascript</li>
+                        </ul>
+                        <a class="button" href="">See project</a>
+                    </section>
+                </section>
+    
+                <section class="card">
```

Comment:
- [ ] Nice work. By the look of things, you included much space between the cards. I will suggest you reduce the space.

Your Design | Figma Design |
:---------------:|:-------------------:|
![e6](https://user-images.githubusercontent.com/39523312/149404636-b6e5d6c2-5a32-48fc-9752-7afbe0d970e5.png) |![e5](https://user-images.githubusercontent.com/39523312/149404632-021acf40-d450-4279-a110-736868982ab7.png)

Reply:
It's intentional design, I think more space between sections looks better when navigating though the page, is just some padding that separates those.